src/pages/320x240x16/trim_page.c

```diff
@@ -64,10 +64,10 @@ static int row_cb(int absrow, int relrow, int y, void *data)
 static void _show_page()
 {
```

Comment:
Lint Error:
```
60:  At least two spaces is best between code and comments  
60:  Should have a space between // and comment  
60:  Use int16/int64/etc, rather than the C type long  
65:  { should almost always be at the end of the previous line  
```